### DISCUSSION TITLE
The airflow dag is triggered but the task status immediately changes to up_for_retry

### DISCUSSION THREAD

**Discussion by admh** [apache/airflow#21384] [General]
### Apache Airflow version

2.0.1

### What happened

The airflow dag is triggered but the task status immediately changes to up_for_retry

We set the number of up_for_retry to 5 times, but after 5 times the dag will fail

When I set the task status in the dag to clear, the task can be executed

This question has been bothering us for a while, hope to get a reply
grateful
![image](https://user-images.githubusercontent.com/19163424/152773046-e1ca5cb5-7f3c-4dee-8e8f-c60eb4610c7e.png)
![image](https://user-images.githubusercontent.com/19163424/152773534-5e5e0c8f-4e89-4791-aa1a-2b6cbbb80085.png)

![image](https://user-images.githubusercontent.com/19163424/152773256-3a73debf-4765-43ec-b100-5c802aa0be08.png)

### Operating System

Container-Optimized OS from Google

### Deployment

Composer

### Deployment details

gcloud composer environments create $ENVIRONMENT_NAME --location=us-east4 --zone=us-east4-a  --disk-size=100GB --machine-type=n2d-standard-4 --node-count=3 --python-version=3 --image-version=composer-1.17.0-airflow-1.10.15  --network=blockchain-vpc --subnetwork=airflow-prod --enable-private-environment --enable-ip-alias --web-server-allow-all

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Until we know it's an issue (and have a reproduction path) - it should be a discussion. I converted it so. Maybe someone will be able to help you but it looks like a race condition somewhat generated by either your DAG or composer environment and your task apparently fails immediately for some reason (but what is the reason is hard to say). 

Not likely to be an Airflow Issue. I suggest to add more debugging to the task that fails and try to figure out why and if you cannot - raise an issue with the composer support. 

But maybe other people who had similar issue will be able to say - I am just guessing as the information you gave is not nearly enough to reason and reproduce the error.
